mappings/net/minecraft/block/entity/BlockEntity.mapping

```diff
@@ -63,7 +63,7 @@ CLASS net/minecraft/unmapped/C_kvegafmh net/minecraft/block/entity/BlockEntity
 		COMMENT Writes this block entity's custom data to the given NBT compound.
 		ARG 1 nbt
 			COMMENT the NBT to write to
-	METHOD m_qrmwzmws onSyncedBlockEvent (II)Z
+	METHOD m_qrmwzmws data (II)Z
```

Comment:
What does this do?

Reply:
changed it to data on mistake, should be called onSyncedBlockEvent